hs/src/Reach/AST/PL.hs

```diff
@@ -256,14 +256,19 @@ instance Pretty a => Pretty (CHandlers_ a) where
   pretty (CHandlers m) =
     render_obj m
 
+data ColorGraphs = ColorGraph {
+  typeGraph :: M.Map DLType Int,
+  varGraph  :: M.Map DLVar Int }
+  deriving (Eq)
+
 data CPProg a
-  = CPProg SrcLoc (CHandlers_ a)
+  = CPProg SrcLoc ColorGraphs (CHandlers_ a)
```

Comment:
I think the new analysis should be totally separate and just return this information

Reply:
Separated this out to just return the graphs from the function